kodilla-basic-tests/src/test/java/com/kodilla/bank/homework/BankTestSuite.java

```diff
@@ -1,6 +1,52 @@
 package com.kodilla.bank.homework;
 
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+
 public class BankTestSuite {
 
+    @Test
+    public void shouldReturnCashMachineBalance() {
+        CashMachine cashMachine = new CashMachine();
+        int[] cashmachines = new int[] {50, 150, 400, -150};
+        int sumBalance = cashMachine.balanceOfCashMachine();
+        assertEquals(450, sumBalance);
+    }
+
+    @Test
+    public void testgetBalance() {
+        Bank bank = new Bank();
```

Comment:
stworzenie obiektu typu Bank to jedno, zaś trzeba w nim dodać jeszcze przecież CashMachine's, w ktorych znów są transakcje.